ducktape/cluster/cluster.py

```diff
@@ -38,7 +46,19 @@ def __len__(self):
         """Size of this cluster object. I.e. number of 'nodes' in the cluster."""
         raise NotImplementedError()
 
-    def request(self, nslots):
+    def request_subcluster(self, num_nodes):
+        """Return an instance of Cluster with the specified num_nodes.
+
+        All implementations of this method within ducktape make use of the num_nodes parameter, but in some
```

Comment:
Hm, yeah I can remove this part of the comment.

I was imagining a cluster implementation where the size of the cluster is unbounded, aka effectively infinite.